.rubocop.yml

```diff
@@ -1,72 +1,332 @@
   Exclude:
-    # Templates are really ERB which Rubocop does not parse
-    - 'lib/generators/rspec/*/templates/**/*'
+    # RSpec is tightly coupled to capybara right now, this should be
+    # re-evaluted in the future. For now we allow the empty rescue
+    - lib/rspec/rails/vendor/capybara.rb
 
-FileName:
+Metrics/AbcSize:
+  Enabled: false
+
+Metrics/BlockLength:
+  Enabled: false
+
+# Warns when the class is excessively long.
+Metrics/ClassLength:
+  Max: 100
+
+Metrics/PerceivedComplexity:
+  Enabled: false
+
+# Who cares what we call the argument for binary operator methods?
+Naming/BinaryOperatorParameterName:
+  Enabled: false
+
+Naming/ConstantName:
+  Enabled: false
+
+Naming/FileName:
   Exclude:
     # We break convention here so that when bundler requires the gem, which
     # uses the gem name, things work without user configuration.
     - lib/rspec-rails.rb
 
-HandleExceptions:
-  Exclude:
-    # RSpec is tightly coupled to capybara right now, this should be
-    # re-evaluted in the future. For now we allow the empty rescue
-    - lib/rspec/rails/vendor/capybara.rb
+Naming/HeredocDelimiterNaming:
+  Enabled: false
+
+Naming/MemoizedInstanceVariableName:
+  Enabled: false
+
+Naming/MethodParameterName:
+  Enabled: false
+
+# We have too many special cases where we allow generator methods or prefer a
+# prefixed predicate due to it's improved readability.
+Naming/PredicateName:
+  Enabled: false
+
+Naming/RescuedExceptionsVariableName:
+  Enabled: false
+
+Naming/VariableNumber:
+  Enabled: false
 
-IfUnlessModifier:
+Style/CollectionMethods:
+  PreferredMethods:
+    reduce: 'inject'
+
+Style/AccessModifierDeclarations:
+  Enabled: false
+
+# "Use alias_method instead of alias"
+# We're fine with `alias`.
+Style/Alias:
+  Enabled: false
+
+Style/BlockDelimiters:
+  Enabled: false
+
+# "Avoid the use of the case equality operator ==="
+# We prefer using `Class#===` over `Object#is_a?` because `Class#===`
+# is less likely to be monkey patched than `is_a?` on a user object.
+Style/CaseEquality:
+  Enabled: false
+
+Style/ClassCheck:
+  Enabled: false
+
+Style/ConditionalAssignment:
+  Enabled: false
+
+Style/DateTime:
+  Enabled: false
+
+# We use YARD to enforce documentation. It works better than rubocop's
+# enforcement...rubocop complains about the places we re-open
+# `RSpec::Expectations` and `RSpec::Matchers` w/o having doc commments.
+Style/Documentation:
+  Enabled: false
+
+Style/DoubleNegation:
+  Enabled: false
+
+Style/EmptyMethod:
+  Enabled: false
+
+Style/EmptyCaseCondition:
+  Enabled: false
+
+Style/EmptyElse:
+  Enabled: false
+
+Style/FormatString:
+  EnforcedStyle: percent
+
+Style/FormatStringToken:
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Style/GuardClause:
+  Enabled: false
+
+Style/IdenticalConditionalBranches:
+  Enabled: false
+
+Style/IfUnlessModifierOfIfUnless:
+  Enabled: false
+
+Style/IfInsideElse:
+  Enabled: false
+
+Style/IfUnlessModifier:
```

Comment:
Previously we disabled this, and we have a bunch of failures now due to it.

Reply:
I was misled by configuration composition.